Holovibes/includes/enum/enum_space_transformation.hh

```diff
@@ -16,19 +16,29 @@ namespace holovibes
  */
 enum class SpaceTransformation
 {
-    NONE = 0, /*!< Nothing Applied */
-    FFT1,     /*!< Fresnel Transform */
-    FFT2      /*!< Angular spectrum propagation */
+    NONE = 0,  /*!< Nothing Applied */
+    FRESNELTR, /*!< Fresnel Transform */
+    ANGULARSP  /*!< Angular spectrum propagation */
 };
 
 // clang-format off
+
+/* There are that many entries here for the purpose of Retrocompatibility.
+ * The original ones were FFT1 and FFT2, but when the names were changed to
+ * FRESNELTR and ANGULARSP, they were left here to allow older version of
+ * compute settings to work with the new names.
+ */
 SERIALIZE_JSON_ENUM(SpaceTransformation, {
     {SpaceTransformation::NONE, "NONE"},
-    {SpaceTransformation::FFT1, "FFT1"},
-    {SpaceTransformation::FFT2, "FFT2"},
-    {SpaceTransformation::FFT1, "1FFT"}, // Compat
-    {SpaceTransformation::FFT2, "2FFT"}, // Compat
-    {SpaceTransformation::NONE, "None"}, // Compat
+    {SpaceTransformation::FRESNELTR, "FRESNELTR"}, // Actual saved strings
```

Comment:
Maybe reorder for visibility like all FFT1 conversion before and then FFT2 instead of having FFT1 FFF2 FFT1 ....